### PR TITLE
Added recipe for glue-geospatial

### PR DESCRIPTION
**Pull request by astrofrog**

### CODE REVIEW

**Comment by astrofrog** [comment]
This should be ready to go!

File: recipes/glue-geospatial/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  version: {{version}}
+
+source:
+  fn: glue-geospatial-{{version}}.tar.gz
+  url: https://pypi.io/packages/source/g/glue-geospatial/glue-geospatial-{{version}}.tar.gz
+  sha256: 95e33a0b317f3df406acffd808112f852b04687b0b1062467e044e99634a30dc
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (mariusvniekerk) [inline recipes/glue-geospatial/meta.yaml]: Can you change this to the new pip variant as in the example

File: recipes/glue-geospatial/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+requirements:
+
+  build:
+    - python
```
Comment (mariusvniekerk) [inline recipes/glue-geospatial/meta.yaml]: Change this to python and pip

File: recipes/glue-geospatial/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+package:
+  name: glue-geospatial
+  version: {{version}}
+
+source:
+  fn: glue-geospatial-{{version}}.tar.gz
+  url: https://pypi.io/packages/source/g/glue-geospatial/glue-geospatial-{{version}}.tar.gz
+  sha256: 95e33a0b317f3df406acffd808112f852b04687b0b1062467e044e99634a30dc
+
+build:
```
Comment (mariusvniekerk) [inline recipes/glue-geospatial/meta.yaml]: Can this be `no-arch: python` ?

File: recipes/glue-geospatial/meta.yaml (hunk 1 above)
Comment (astrofrog) [reply recipes/glue-geospatial/meta.yaml]: Fixed

File: recipes/glue-geospatial/meta.yaml (hunk 2 above)
Comment (astrofrog) [reply recipes/glue-geospatial/meta.yaml]: Fixed

File: recipes/glue-geospatial/meta.yaml (hunk 3 above)
Comment (astrofrog) [reply recipes/glue-geospatial/meta.yaml]: Fixed

**Pull request merged**